README.md

```diff
@@ -41,7 +41,9 @@ To visualise data, you must have an ESP project running in either SAS Event Stre
 
 ### Examples
 
-Some SAS Event Stream Processing Studio examples include Grafana dashboards:
+Some SAS Event Stream Processing Studio examples include Grafana dashboards.
+
+**_NOTE:_** The dashboards have been created using a standalone version of ESP running in the same namespace as ESP. If you are using a different environment, like Viya, you will need to recreate the queries becasue the connection URLs will differ.    
```

Comment:
Here are my suggested changes. I'll post further info in Jira.
> **Note**: The dashboards have been created using standalone SAS Event Stream Processing that is running in the same namespace as Grafana. If you are using a different environment, such as the SAS Viya platform, you must recreate the queries because the connection URLs will differ.